### ISSUE TITLE
Issue following the display_plugin refactoring

### ISSUE THREAD

**Issue by nicolargo** [nicolargo/glances#2949]
Some time, when Glances starts on my system, i have the following error:

```
es/standalone.py", line 186, in serve_forever
                                                 while self.__serve_once():
                                                                             File "/home/nhe/dev/glances/glances/standalone.py", line 158, in __serve_once
                                         ret = not self.screen.update(self.stats, duration=adapted_refresh)
                                                                                                             File "/home/nhe/dev/glances/glances/outputs/glances_curses.py", line 1105, in update
                                                                                self.flush(stats, cs_status=cs_status)
   File "/home/nhe/dev/glances/glances/outputs/glances_curses.py", line 1086, in flush
                                                                                          self.display(stats, cs_status=cs_status)
               File "/home/nhe/dev/glances/glances/outputs/glances_curses.py", line 555, in display
                                                                                                       self.__display_right(__stat_display)
                        File "/home/nhe/dev/glances/glances/outputs/glances_curses.py", line 797, in __display_right
                                                                                                                        self.display_plugin(
                         File "/home/nhe/dev/glances/glances/outputs/glances_curses.py", line 979, in display_plugin
                                                                                                                        y, x, x_max = self.display_msg(plugin_stats, init, helper)
                                                               File "/home/nhe/dev/glances/glances/outputs/glances_curses.py", line 1017, in display_msg
                                       x, x_max = self.display_stats_with_current_size(m, y, x, x_max)
                                                                                                      TypeError: cannot unpack non-iterable NoneType object
```

Introduced in https://github.com/nicolargo/glances/pull/2932

@ariel-anieli can you have a look ?

**Comment by ariel-anieli** [comment]
Sure, @nicolargo; I'm looking at it.

**Comment by ariel-anieli** [comment]
Fixed in 4bcbc12ceae3e5d36b532d10397ae6b1ad4e5914.
